db/blip_handler_collections.go

```diff
@@ -85,10 +85,32 @@ func (bh *blipHandler) handleGetCollections(rq *blip.Message) error {
 		}
 		delete(value, BodyId)
 		checkpoints[i] = value
+		// Should we add this if connection is nil?
+		bh.collectionMapping = append(bh.collectionMapping, &Database{
+			DatabaseContext: collection.CollectionCtx,
+			user:            bh.blipContextDb.User(), Ctx: bh.db.Ctx,
+		})
```

Comment:
I will need to assert bh.collectionMapping is the same length as checkpoints returned, which means I need to append the bh.collectionMapping in the nil to empty body cases.

Reply:
I can't easily do this until CBG-2204, so deferring this.